src/client/components/app/VaultDetail/VaultDetailPanels.tsx

```diff
@@ -286,7 +298,21 @@ export const VaultDetailPanels = ({
 
               <InfoValueRow>
                 <span>{t('vaultdetails:overview-panel.apy')}</span>
-                <StyledText fontWeight="bold">{formatApy(selectedVault.apyData, selectedVault.apyType)}</StyledText>
+                <TextWithIcon>
+                  <StyledText fontWeight="bold">
+                    <span>{formatApy(selectedVault.apyData, selectedVault.apyType)}</span>
+                  </StyledText>
+                  {!isApyNewOrNA(selectedVault.apyType) && selectedVault.apyMetadata && (
```

Comment:
Would prefer not using conditional and ternary logic in JSX. Put this in a variable intead:

const showTooltip = !isApyNewOrNA(selectedVault.apyType) && selectedVault.apyMetadata;
const toolTip = (...jsx);

<TextWithIcon>...{toolTip}...</TextWithIcon>